index.html

```diff
@@ -2,13 +2,138 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
+    <meta name="description" content="Bocharov design studio">
+    <meta name="keywords" content="interior design, author's works">
+    <meta name="autor" content="Anton and Irina Bocharov">
+    <meta name="vienport" content="width=device-width, initial-scale=1.0">
     <title>Моя первая страница</title>
   </head>
   <body>
     <h1>Моя первая страница</h1>
-    <hr />
+    <hr>
     <p>
       <a href="Hello/about.html">об авторе</a>
     </p>
+    <img src="./images/flower.jpg" alt="flowers">
+    <b>
+      <ol>
+        <li>Знакомство с Web</li>
+        <ul>
+          <li>Краткая история Web и что такое интернет.</li>
+          <li>Как устроен браузер и как он получает страницы по сети.</li>
+          <li>Что такое Git?</li>
+          <li>Создание первого репозитория.</li>
+          <li>О файле .gitignore.</li>
+        </ul>
+        <li>HTML-верстка, первые шаги</li>
+        <ul>
+          <li>Настройка рабочей среды.</li>
+          <li>Краткая история HTML.</li>
+          <li>Какие бывают теги и что такое атрибут.</li>
+          <li>Теги для работы с текстом.</li>
+          <li>Создание собственной страницы.</li>
+        </ul>
+        <li>Погружение в HTML-разметку</li>
+        <ul>
+          <li>Мета-теги.</li>
+          <li>Мнемоники.</li>
+          <li>Тег для работы с изображениями.</li>
+          <li>Теги для работы со списками.</li>
+          <li>Теги для работы с таблицами.</li>
+          <li>Теги для работы с формами.</li>
+          <li>Div-блоки.</li>
+        </ul>
+      </ol>
+    </b>
+    <table border="1" width="100%">
+      <thead>
+      <tr>
+        <th>Tag</th>
+        <th>Description</th>
+      </tr>
+      </thead>
+      <tbody>
+      <tr>
+        <td>&#60;table&#62;</td>
+        <td>Defines a table</td>
+      </tr>
+      <tr>
+        <td>&#60;tr&#62;</td>
+        <td>Defines a row in table</td>
+      </tr>
+      <tr>
+        <td>&#60;th&#62;</td>
+        <td rowspan="2">Defines a "header cell"/"cell" in table</td>
+      </tr>
+      <tr>
+        <td>&#60;td&#62;</td>
+      </tr>
+      <tr>
+        <td colspan="2">Rarely used</td>
+      </tr>
+      <tr>
+        <td>&#60;caption&#62;</td>
+        <td>Defines a table caption</td>
+      </tr>
+      <tr>
+        <td>&#60;thead&#62;</td>
+        <td>Groups the header content in a table</td>
+      </tr>
+      <tr>
+        <td>&#60;tbody&#62;</td>
+        <td>Groups the body content in a table</td>
+      </tr>
+      <tr>
+        <td>&#60;tfood&#62;</td>
+        <td>Groups the footer content in a table</td>
+      </tr>
+      </tbody>
+    </table>
+    <form>
+      <fieldset>
+        <legend>My first form</legend>
+        <p>
```

Comment:
The same as `<b>`. Use `<div>` to wrap the blocks.

Reply:
Do I need to replace <p>?

Reply:
I corrected.